package.json

```diff
@@ -4,15 +4,14 @@
   "main": "dist/index.js",
```

Comment:
After adding one more folder with typescript files (which is "test" in our case), value here must be `dist/src/index.js`